app/src/main/java/com/dessalines/thumbkey/keyboards/ENHRMessagEase.kt

```diff
@@ -0,0 +1,566 @@
+                                                                action = KeyAction.CommitText(":"),
+                                                                color = ColorVariant.MUTED,
+                                                        ),
+                                                SwipeDirection.BOTTOM to
+                                                        KeyC(
+                                                                display = KeyDisplay.TextDisplay("."),
+                                                                action = KeyAction.CommitText("."),
+                                                                color = ColorVariant.MUTED,
+                                                        ),
+                                                SwipeDirection.BOTTOM_LEFT to
+                                                        KeyC(
+                                                                display = KeyDisplay.TextDisplay(","),
+                                                                action = KeyAction.CommitText(","),
+                                                                color = ColorVariant.MUTED,
+                                                        ),
+                                        ),
+                                ),
+                                KeyItemC(
+                                        center =
+                                        KeyC(
+                                                display = KeyDisplay.TextDisplay("S"),
+                                                action = KeyAction.CommitText("S"),
+                                                size = FontSizeVariant.LARGE,
+                                                color = ColorVariant.PRIMARY,
+                                        ),
+                                        swipeType = SwipeNWay.FOUR_WAY_DIAGONAL,
+                                        swipes =
+                                        mapOf(
+                                                SwipeDirection.TOP_LEFT to
+                                                        KeyC(
+                                                                display = KeyDisplay.TextDisplay("F"),
+                                                                action = KeyAction.CommitText("F"),
+                                                        ),
+                                        ),
+                                ),
+                                BACKSPACE_KEY_ITEM,
+                        ),
+                        listOf(
+                                SPACEBAR_KEY_ITEM,
+                                RETURN_KEY_ITEM,
+                        ),
+                ),
+        )
+
+val KB_EN_HR_MESSAGEASE: KeyboardDefinition =
+        KeyboardDefinition(
+                title = "english messagease with croatian guillemets",
```

Comment:
Look at the other keyboard names. languages first, and in their original characters. Then the type of keyboard last.